src/promptflow/promptflow/contracts/tool.py

```diff
@@ -254,7 +255,7 @@ def serialize(self) -> dict:
         """
 
         data = {}
-        data["type"] = ([t.value for t in self.type],)
+        data["type"] = [t.value for t in self.type]
```

Comment:
Why updating this, is it by design?

Reply:
This data type is not compatible with the deserialize function. According to the logic in deserialize function, there should not be brackets outside [t.value for t in self.type].